src/pages/Home.js

```diff
@@ -36,8 +32,65 @@ export default function Home() {
           text="SHOW ME THE BEST TIME"
         />
       </Link>
+
       <RedditTopic />
+
       <HeatMap />
+
+      <StyledInfo>
+        <StyledTitle>
+          How it works
+        </StyledTitle>
+
+        <StyledParagraph>
+          <StyledText>
+            • We find the 500 top posts from the past year for a subreddit.
+          </StyledText>
+          <StyledText>
+            • The data is visualized in a heatmap grouped by weekday and hour of the day.
+          </StyledText>
+          <StyledText>
+            • See immediately when to submit your reddit post.
+          </StyledText>
+        </StyledParagraph>
+      </StyledInfo>
+
+      <StyledInfo>
```

Comment:
You could extract this component into a separate file with props `title` and `children`. But that's just a suggestion

Reply:
I'm having problems passing the link inside the text on the children.